### ISSUE TITLE
Review version policy WindowsAzure.Storage

### ISSUE THREAD

**Issue by yvesgoeleven** [Particular/NServiceBus.Azure#299]
Current: Fixed Major Range

Used in 
- NServiceBus.Azure.Transports.WindowsAzureStorageQueues: Range [8.0.0, 9.0.0)
- NServiceBus.Persistence.AzureStorage: Range [8.0.0, 9.0.0)
- NServiceBus.DataBus.AzureBlobStorage: Range [8.0.0, 9.0.0)
- NServiceBus.Hosting.Azure: Range [7.0.0, 9.0.0)

**Comment by yvesgoeleven** [comment]
Suggestion, Fixed Major Range [7.0.0, 9.0.0) for all

**Comment by SeanFeldman** [comment]
For all? Including NServiceBus.Azure.Transports.WindowsAzureStorageQueues and NServiceBus.Persistence.AzureStorage that are currently have lower version as 8.0?

**Comment by yvesgoeleven** [comment]
yeah I would treat all the same, packages are used together if the rules vary customers will get into conflicts (e.g; newtonsoft support case from last week)

**Comment by SeanFeldman** [comment]
Sounds good.

**Comment by Scooletz** [comment]
As we know the API from the past and our usage of it: 👍

**Comment by danielmarbach** [comment]
Again too dangerous. A support case doesn't justify the risk we are exposing ourselves here. So 👎

**Comment by yvesgoeleven** [comment]
@danielmarbach what risk? we have been running with version 7 like forever, it's just the start of the range that moves back to a version that we are very familiar with

**Comment by danielmarbach** [comment]
For a generic thing like

NServiceBus.Hosting.Azure

I see now risk. But this issues states that we want [7.0.0, 9.0.0) for all the above-stated ones. 

A transport or a persistence is likely to make usage of a specific feature of a major version of that library for optimization purpose or to expose some hook of whatever kind. As soon as you do things start to break. Which has severe and more consequences then us having to remember to release a minor from time to time.
